.rtlcssrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This is the default config as mentioned at https://rtlcss.com/learn/usage-guide/cli/

Without a config file there are warnings.